contacts.html

```diff
@@ -24,21 +24,24 @@ <h2>Popular science magazine OYLA</h2>
             <h3>Contact Us</h3>
             <h4>Information about the publisher</h4>
             <address>
-                <p>Popular science magazine OYLA is owned and operated by OYLA MAGAZINE LTD (COMPANY NUMBER: 12718869).<br>128 City Road, London, EC1V 2NX, United Kingdom.<br><br><strong>Tel:</strong> +44 20 4577 04 07<br><strong>Working hours:</strong> 9-5pm, Mon-Fri</p>
+                <p>Popular science magazine OYLA is owned and operated by OYLA MAGAZINE LTD (COMPANY NUMBER: 12718869).<br>128 City Road, London, EC1V 2NX, United Kingdom.<br><br><strong>Tel:</strong> +44 20 4577 04 07<br><strong>Working hours:</strong> 9-5pm, Mon-Fri
+                </p>
             </address>
         </article>
         <hr>
         <article>
             <h4>Subscriptions issues</h4>
-            <p>If you have a query about a subscription or back issue (concerning a new order, an existing order or a change of address or other details), please contact<br> our customer services team:<br><strong>Your personal account:</strong><br><br><strong>E-mail:</strong>subscription@oyla.uk<br><strong>Phone:</strong> +44 20 4577 04 07<br><strong>The fastest way to contact us is to message: </strong>Messenger </p>
+            <p>If you have a query about a subscription or back issue (concerning a new order, an existing order or a change of address or other details), please contact<br> our customer services team:<br><strong>Your personal account:</strong><br><br><strong>E-mail:</strong>subscription@oyla.uk<br><strong>Phone:</strong> +44 20 4577 04 07<br><strong>The fastest way to contact us is to message: </strong>Messenger
```

Comment:
Same as in previous comment, closing p tag is repositioned, but the content of the element needs improvement